### PR TITLE
[Ubuntu] Pin PostgreSQL version installed in the toolset

### PR DESCRIPTION
**Pull request by shilovmaksim**
# Description
We need to move PostgreSQL along with some other tools to the toolset on all OSes to make versions control more granular.
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/postgresql.sh [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+##  Desc:  Installs PostgreSQL
 ################################################################################
 
 REPO_URL="https://apt.postgresql.org/pub/repos/apt/"
 
-#Preparing repo for PostgreSQL 12.
+# Preparing repo for PostgreSQL
 wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-echo "deb $REPO_URL `lsb_release -cs`-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+echo "deb $REPO_URL $(getOSVersionLabel)-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
```
Comment (al-cheb) [inline images/linux/scripts/installers/postgresql.sh:11]: You should import this function before using from a helper script.

File: images/linux/scripts/tests/Databases.Tests.ps1 [hunk 2]
```diff
@@ -9,12 +9,19 @@ Describe "MongoDB" {
...
     It "PostgreSQL Service" {
         "sudo systemctl start postgresql" | Should -ReturnZeroExitCode
         (Get-CommandResult "pg_isready").Output | Should -Be "/var/run/postgresql:5432 - accepting connections"
         "sudo systemctl stop postgresql" | Should -ReturnZeroExitCode 
     }
+
+    It "PostgreSQL version should correspond to the version in the toolset" {
+        $toolsetVersion = (Get-ToolsetContent).postgresql.version
+        # Client version
+        (&psql --version).split(" ")[-1] | Should -BeLike "$toolsetVersion*"
```
Comment (al-cheb) [inline images/linux/scripts/tests/Databases.Tests.ps1:21]: nitpick: remove &

File: images/linux/scripts/installers/postgresql.sh (hunk 1 above)
Comment (shilovmaksim) [reply images/linux/scripts/installers/postgresql.sh:15]: Thanks. Done

File: images/linux/scripts/tests/Databases.Tests.ps1 (hunk 2 above)
Comment (shilovmaksim) [reply images/linux/scripts/tests/Databases.Tests.ps1]: Updated

**Review by miketimofeev — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Pull request merged**